custom/icds_reports/migrations/0176_child_health_mother_id_birth_weight.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.26 on 2019-12-04 05:51
+from __future__ import unicode_literals
+
+from django.db import migrations
+from corehq.sql_db.operations import RawSQLMigration
+
+
+migrator = RawSQLMigration(('custom', 'icds_reports', 'migrations', 'sql_templates', 'database_views'))
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0175_bihar_api_view'),
+    ]
+
+    operations = [
```

Comment:
@challabeehyv  you will also have to rebuild the child_helath_monthly view. As this is a distributed table and when a distributed table is altered its underlying view also needs to be rebuilt